app/controllers/concerns/idv_step_concern.rb

```diff
@@ -111,9 +111,7 @@ def flow_policy
 
   def confirm_step_allowed
     # set it everytime, since user may switch SP
-    idv_session.selfie_check_required =
-      FeatureManagement.idv_allow_selfie_check? &&
-      resolved_authn_context_result.biometric_comparison?
+    idv_session.selfie_check_required = resolved_authn_context_result.biometric_comparison?
```

Comment:
A note: This is accessed in a lot of places where `resolved_authn_context_result` is available. I think we could go back and clean those up at some point.

This seems to be set in the session so we can access it in the `preconditions` block for `.step_info` for a number of controllers.